### PR TITLE
Documentation on using fetch-forms-from-google-drive

### LINKED ISSUE #84: Missing instructions on secrets files for fetching-forms-from-google-drive
https://github.com/medic/medic-docs/blob/9fe07f18f4eaeb3ff165bf1233bc7f4b517ff58d/configuration/update-standard-forms.md

"The files to download are listed in forms-on-google-drive.json. Use the fetch-forms-from-google-drive action in medic-conf to get the forms, then follow the instructions to authenticate accordingly."

Running `medic-conf fetch-forms-from-google-drive` yields:

```
WARN Error reading file: ./.gdrive.secrets.json 
WARN Error parsing JSON in: ./.gdrive.secrets.json 
INFO Failed to load google drive secrets from file. Error: ENOENT: no such file or directory, open './.gdrive.secrets.json'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at read (/usr/lib/node_modules/medic-conf/src/lib/sync-fs.js:11:15)
    at Object.readJson (/usr/lib/node_modules/medic-conf/src/lib/sync-fs.js:29:23)
    at oauthClient (/usr/lib/node_modules/medic-conf/src/lib/google-auth.js:47:21)
    at module.exports (/usr/lib/node_modules/medic-conf/src/lib/google-auth.js:12:18)
    at module.exports (/usr/lib/node_modules/medic-conf/src/lib/fetch-files-from-google-drive.js:8:10)
    at Object.execute (/usr/lib/node_modules/medic-conf/src/fn/fetch-forms-from-google-drive.js:7:11)
    at executeAction (/usr/lib/node_modules/medic-conf/src/lib/main.js:198:40)
    at module.exports (/usr/lib/node_modules/medic-conf/src/lib/main.js:188:11) 
WARN Missing .client_id in ./.gdrive.secrets.json or env var GOOGLE_CLIENT_ID! 
WARN Missing .client_secret in ./.gdrive.secrets.json or env var GOOGLE_CLIENT_SECRET! 
WARN Missing .redirect_uris in ./.gdrive.secrets.json or env var GOOGLE_REDIRECT_URI! 
ERROR Error: Missing required config for google drive access.  Please check warnings.
    at oauthClient (/usr/lib/node_modules/medic-conf/src/lib/google-auth.js:64:27)
    at module.exports (/usr/lib/node_modules/medic-conf/src/lib/google-auth.js:12:18)
    at module.exports (/usr/lib/node_modules/medic-conf/src/lib/fetch-files-from-google-drive.js:8:10)
    at Object.execute (/usr/lib/node_modules/medic-conf/src/fn/fetch-forms-from-google-drive.js:7:11)
    at executeAction (/usr/lib/node_modules/medic-conf/src/lib/main.js:198:40)
    at module.exports (/usr/lib/node_modules/medic-conf/src/lib/main.js:188:11)
    at /usr/lib/node_modules/medic-conf/src/bin/medic-conf.js:11:24
    at Object.<anonymous> (/usr/lib/node_modules/medic-conf/src/bin/medic-conf.js:20:3)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10) 
```

This error is not prescriptive or informative. I'm unclear if this is a documentation error or a product issue in medic-conf.

### PR DESCRIPTION
**Pull request by newtewt**
Should close #84  and #78 . Let me know if you feel like any of this is lacking! 

Details the files needed and how to create them.

### CODE REVIEW

File: content/en/apps/guides/forms/google-docs.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+This action requires two files in the folder you are going to execute this command from. The files are `forms-on-google-drive.json` and  `.gdrive.secrets.json`. 
+
+
+
+## forms-on-google-drive.json 
+
+This file should be a json file with a key value pair. The key being the folder/name of the doc when it gets saved locally. The value being the ID from the document saved in google drive. 
+
+To get the ID for the form. Open the form in google sheets. The URL will look like this https://docs.google.com/spreadsheets/d/12345ABCDEF/edit#gid=555666888 . The ID is the middle portion `12345ABCDEF`
```
Comment (abbyad) [inline content/en/apps/guides/forms/google-docs.md:20]: We should make sure to be consistent with capitalization of proper nouns.

```suggestion
To get the File ID for the form open the form in Google Sheets. The URL will look something like `https://docs.google.com/spreadsheets/d/12345ABCDEF/edit#gid=555666888` . The ID is the middle portion `12345ABCDEF`
```

File: content/en/apps/guides/forms/google-docs.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+```json
+{
+  "app/sample_report.xlsx": "12345ABCDEF",
+  "contact/person-create.xlsx": "ABCDEF123456",
+}
+```
+
+## .gdrive.secrets.json
+
+This file contains authentication secrets that are generated by enabling access to your drive using these [instructions](https://developers.google.com/drive/api/v3/about-auth)
```
Comment (abbyad) [inline content/en/apps/guides/forms/google-docs.md:31]:
```suggestion
This file contains authentication secrets that are generated by enabling access to your Google Drive using these [instructions](https://developers.google.com/drive/api/v3/about-auth).
```

File: content/en/apps/guides/forms/google-docs.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+```
+
+## Run the Medic Conf Action
+
+With the files in place. Run `medic-conf fetch-forms-from-google-drive`
+
+Medic-conf will then prompt for an access code from your authenticated user. Follow the prompts through google's authentication and copy the access code into your command line. The forms will then be copied to the forms folder in the directory you are executing from. 
+
+*Note: If the browser does not launch own its own. Copy the URL and paste the authentication url into a browser and then follow the prompts.*
```
Comment (abbyad) [inline content/en/apps/guides/forms/google-docs.md:53]:
```suggestion
## Run

With the files in place, run medic-conf's `fetch-forms-from-google-drive` action in your CHT app's top level directory.

```sh
medic-conf fetch-forms-from-google-drive.
```

You will be prompted for an access code from your authenticated user. Follow the prompts through Google's authentication page and copy the access code into your command line. The forms will then be downloaded to the locations set in the `forms-on-google-drive.json` file. 

{{% alert title="Note" %}} 
If the browser does not launch own its own copy the URL and paste the authentication url into a browser and then follow the prompts.
{{% /alert %}}
```

**Review by abbyad — CHANGES_REQUESTED** [review]
Great to have this added, nice work @newtewt!

Some text and styling edits below. We should also rename the file to the more generic Google Drive, rather than Docs.

**Comment by newtewt** [comment]
@abbyad  updated!

**Review by abbyad — APPROVED** [review]
Looks great!

**Pull request merged**